pkg/ccl/backupccl/backuppb/backup.go

```diff
@@ -226,11 +231,46 @@ func (e *ExportStats) Combine(other bulk.TracingAggregatorEvent) {
 	}
 }
 
+// String implements the AggregatorEvent and stringer interfaces.
+func (e *ExportStats) String() string {
+	const mb = 1 << 20
+	var b strings.Builder
+	if e.NumFiles > 0 {
+		b.WriteString(fmt.Sprintf("num_files: %d\n", e.NumFiles))
```

Comment:
It doesn't matter, but if you wanted, `b.WriterString(fmt.Sprintf(` -> `fmt.Fprintf(b`.

Reply:
I've left this as is because `fmt.Fprintf` leaves a yellow squigly saying unhandled error. We do have precedence for this elsewhere so if this suggestion is for something more than code-style I am happy to make the change.

Reply:
Fine as is!